godtools/Managers/FollowUpsManager.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  FollowUpsManager.swift
+//  godtools
+//
+//  Created by Pablo Marti on 6/8/17.
+//  Copyright © 2017 Cru. All rights reserved.
+//
+
+import UIKit
+import PromiseKit
+
+class FollowUpsManager: GTDataManager {
+    
+    let path = "follow_ups"
+    
+    func createSubscriber(params: [String: String]) -> Promise<Void>? {
+        showNetworkingIndicator()
```

Comment:
Is the networking indicator being hidden somewhere else? I don't see it in this PR.